src/controllers/accommodation.js

```diff
@@ -0,0 +1,52 @@
+import models from '../db/models';
+import dotenv from 'dotenv';
+dotenv.config();
```

Comment:
please add two empty lines on after imports.

Reply:
I have implemented it, for now, you can check

Reply:
I resolved the issue by configuring my eslint